README.rst

```diff
@@ -131,6 +131,17 @@ Stingray uses `py.test` for testing. To run the tests, try::
 
```

Comment:
Now that we are at it, we might want to add the test dependencies, like `pytest-astropy`

Reply:
I'm not sure I understand this request: add the test dependencies where? In the documentation? Can you expand a little on what you mean?

Reply:
Right, sorry :).
I meant that we might want to specify, in the instructions for testing, that additional dependencies are required to run them

Reply:
Are those dependencies not automatically installed when installing stingray?

Reply:
Not necessarily. For example, you don't need pytest-astropy to run stingray